src/libraries/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.byte.cs

```diff
@@ -180,6 +180,13 @@ public void CopyTo_ThrowsWhenSourceLargerThenDestination()
             });
         }
 
+        [Fact]
+        public void End_EqualToGetPositionSize()
```

Comment:
could you please add a similar test for `Start`? And one for all positions from `Start+1` to `End - 1` (inclusive) that verifies that they are not equal to `.Start` and `.End`?